test/spec/features/popup-menu/PopupMenuSpec.js

```diff
@@ -2191,8 +2216,8 @@ function getGroup(groupName) {
   return domQuery('[data-group="' + groupName + '"]', getPopupContainer());
 }
 
-function whenStable() {
-  return new Promise(resolve => setTimeout(resolve, 300));
+function sleep(ms = 300) {
```

Comment:
I'm not sure this rename is sensible. At the end we want to wait until things (preact for that matter) is updated (or stable to inspect).

Reply:
I found `whenStable` really confusing. I will revert that change.